scenario-compiler/src/main/java/pl/touk/nussknacker/engine/spel/internal/ExtensionAwareMethodsDiscovery.java

```diff
@@ -0,0 +1,52 @@
+package pl.touk.nussknacker.engine.spel.internal;
+
+import pl.touk.nussknacker.engine.extension.Extension;
+import pl.touk.nussknacker.engine.extension.ExtensionMethods;
+import pl.touk.nussknacker.engine.extension.ExtensionRuntimeApplicable;
+import scala.collection.JavaConverters;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ExtensionAwareMethodsDiscovery {
```

Comment:
Why is it in java? Can you you left some comment?

Reply:
Moved to scala